src/components/Q&A/AnswerComp.jsx

```diff
@@ -91,25 +86,38 @@ class AnswerComp extends React.Component {
     this.sortByHelp();
   }
 
-  handleSeeMoreAns(ansArr) {
+  handleSeeMoreAns(ansArr, e) {
+    handleInteractions(e, 'Q&A');
     this.setState({ moreBtnClick: true });
     this.setState({ answerArr: ansArr });
   }
-  handleSeeLessAns(ansArr) {
+  handleSeeLessAns(ansArr, e) {
+    handleInteractions(e, 'Q&A');
     this.setState({ moreBtnClick: false });
     var tempArr = ansArr.slice(0, 2);
     this.setState({ answerArr: tempArr });
   }
 
+
   componentDidMount() {
-    this.sortByHelp();
+    if (this.props.quesID !== undefined) {
+        axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/qa/questions/${this.props.quesID}/answers`,
+        {
+          headers: { Authorization: `${API_KEY}` },
+          params: { count: 5, page: 1, question_id: this.props.quesID },
+        }
+      )
+      .then((res) => this.setState({answersFromGet: res.data}))
+      .catch((err) => console.log(err));
+      }
+    setTimeout(this.sortByHelp, 500);
```

Comment:
just curious why this needed a setTimeout? let's not change it, but just a question.